packages/@orchidui-vue/src/Elements/PageTitle/OcPrimaryActions.vue

```diff
@@ -55,14 +58,24 @@ const copyToClipBoard = () => {
       <template #menu>
         <div class="flex flex-col">
           <div class="p-2 border-b border-gray-200">
-            <DropdownItem icon="pencil" text="Customize link" />
+            <DropdownItem
+                icon="pencil"
+                text="Customize link"
+                v-bind="primaryActions?.dropdownOptions[0]"
+                @click="emit('click:primaryActionsDropdown', primaryActions?.dropdownOptions[1])"
+            />
             <DropdownItem
```

Comment:
WDYT about change logic into like this it will do this component more flexible to configure

```
   <template #menu>
        <div class="flex flex-col">
          <div class="p-2 border-b border-gray-200">
            <template
              v-for="(item, i) in primaryActions.dropdownOptions.top"
              :key="i"
            >
              <DropdownItem
                v-if="item.isCopyButton"
                :icon="isCopied ? 'check' : 'copy'"
                :text="isCopied ? 'Link copied!' : 'Copy link'"
                :icon-classes="isCopied ? '!text-oc-success' : ''"
                @click="copyToClipBoard"
              />
              <DropdownItem v-else v-bind="item" />
            </template>
          </div>
          <div class="p-2">
            <DropdownItem
              v-for="(item, i) in primaryActions.dropdownOptions.bottom"
              :key="i"
              v-bind="item"
            />
          </div>
        </div>
      </template>
```